bigquery_etl/cli/utils.py

```diff
@@ -25,10 +25,14 @@ def is_valid_file(ctx, param, value):
     return value
 
 
-def is_authenticated(project_id="moz-fx-data-shared-prod"):
```

Comment:
The Airflow tasks seem to run in the `derived` project. So this check fails

Reply:
It looks like gke_command by default uses `project_id=GoogleCloudBaseHook(gcp_conn_id=gcp_conn_id).project_id` which gives derived-datasets because the GKE cluster still lives there. We should be launching these in shared-prod, so we can add an explict `project_id` param to these tasks in the DAG.

It looks like we updated `bigquery_etl_query` to use shared-prod by default, but not this function.